contracts/SimpleBank.sol

```diff
@@ -91,10 +95,11 @@ contract SimpleBank {
       // return the user's balance.
 
       // 1. Use a require expression to guard/ensure sender has enough funds
-
+      require(balances[msg.sender] >= withdrawAmount);
       // 2. Transfer Eth to the sender and decrement the withdrawal amount from
       //    sender's balance
-
+      balances[msg.sender] -= withdrawAmount;
```

Comment:
There is a missing action which lets the owner get Ether back. A related issue is opened.